neo/bin/export.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+from neo.Settings import settings
+from neo.Implementations.Blockchains.LevelDB.LevelDBBlockchain import LevelDBBlockchain
+from neo.Core.Blockchain import Blockchain
+import argparse
+from tqdm import trange
+import binascii
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-m", "--mainnet", action="store_true", default=False,
+                        help="use MainNet instead of the default TestNet")
+    parser.add_argument("-c", "--config", action="store", help="Use a specific config file")
+
+    # Where to store stuff
+    parser.add_argument("--datadir", action="store",
+                        help="Absolute path to use for database directories")
+
+    parser.add_argument("-o", "--output", help="Where to save output file")
+
+    parser.add_argument("-t", "--totalblocks", help="Total blocks to export", type=int)
+
+    args = parser.parse_args()
+
+    if args.mainnet and args.config:
+        print("Cannot use both --config and --mainnet parameters, please use only one.")
+        exit(1)
+
+    # Setting the datadir must come before setting the network, else the wrong path is checked at net setup.
+    if args.datadir:
+        settings.set_data_dir(args.datadir)
+
+    # Setup depending on command line arguments. By default, the testnet settings are already loaded.
+    if args.config:
+        settings.setup(args.config)
+    elif args.mainnet:
+        settings.setup_mainnet()
+
+    file_path = None
+
+    if not args.output:
+        raise Exception("Please specify an output path")
+    file_path = args.output
+
+    # Instantiate the blockchain and subscribe to notifications
+    blockchain = LevelDBBlockchain(settings.chain_leveldb_path)
+    Blockchain.RegisterBlockchain(blockchain)
+
+    chain = Blockchain.Default()
+
+    file = open(file_path, 'wb')
```

Comment:
I'd like to suggest using `with open(path) as f:` because the current implementation will not close the file if an exception occurs.